packages/utils/src/encodings.ts

```diff
@@ -143,3 +144,52 @@ export function base58ToBytes(s: string): Uint8Array {
 export function bytesToBase58(byteArray: Uint8Array): string {
   return u8a.toString(byteArray, 'base58btc')
 }
+
+/**
+ * Converts a multibase string to the Uint8Array it represents.
+ * 
+ * @param s - the string to be converted
+ * 
+ * @throws if the string is not formatted correctly.
+ * 
+ * @public
+*/
+export function multibaseToBytes(s: string): Uint8Array {
```

Comment:
https://www.w3.org/community/reports/credentials/CG-FINAL-di-eddsa-2020-20220724/#ed25519verificationkey2020

So according to the Ed25519VerificationKey2020 definition, this is correct. But this function assumes that it's a Ed25519VerificationKey2020  publicKeyMultibase, when it should probably take in the key type as a parameter so it can use different multicodec prefix (if needed) for other key types.

As far as I can tell, X25519KeyAgreementKey2020 has no definition, but we should assume it uses the `x25519-pub` multicodec prefix (0xec). I'd probably be ok saying the X25519 key can use either the x25519-pub OR ed25519-pub prefixes (at least in multibaseToBytes), since I've seen a few X25519 keys that were incorrectly constructed that way.

But bytesToMultibase also needs to be refactored to take in a keyType so we know which multicodec prefix to prepend.